engine/src/juliabox/plugins/auth_github/github_auth.py

```diff
@@ -58,8 +65,12 @@ def get(self):
         if code is not False:
             user = yield self.get_authenticated_user(redirect_uri=self_redirect_uri, code=code)
             user_info = yield self.get_user_info(user)
-            self.update_user_profile(user_info)
             user_id = user_info['email']
+            if not user_id:
```

Comment:
It seems to me that `user_info['email']` would throw `KeyError` if email is not set in it.
Should this instead be `if 'email' in user_info`?